presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -603,6 +603,16 @@ std::unique_ptr<common::Filter> toFilter(
       return combineIntegerRanges(bigintFilters, nullAllowed);
     }
 
+    if (type->kind() == TypeKind::DATE) {
```

Comment:
Can we use the `if` block above that handles integers?

Reply:
I suppose we can, however, there would still be a if-else statement because we need to use `dateRangeToFilter` instead of `bigintRangeToFilter`.

Reply:
I missed the `dateRangeToFilter` difference. This is fine. We can remove this when Date becomes a logical type.